src/HRXMLDocumentCreator.php

```diff
@@ -121,25 +121,24 @@ public function createFromTransaction(Transaction $transaction): HRXMLDocument
             $jobPositionInformation->appendChild($jobPositionDescription);
 
             $jobPositionRequirements = $hrxml->createElement('JobPositionRequirements');
+            $qualificationsRequired = $hrxml->createElement('QualificationsRequired');
             foreach ($arbetsformedlingenJob->getQualifications() as $qualificationModel) {
-                $qualificationsRequired = $hrxml->createElement('QualificationsRequired');
                 $qualification = $hrxml->createElement('Qualification');
                 if ($qualificationModel->getType() == Qualification::TYPE_LICENSE) {
                     $qualification->setAttributeNodeNS(new \DOMAttr('type', 'license'));
                     $qualification->setAttributeNodeNS(new \DOMAttr('description', 'DriversLicense'));
+                    $qualification->setAttributeNodeNS(new \DOMAttr('category', $qualificationModel->getCategory()));
                 } elseif ($qualificationModel->getType() == Qualification::TYPE_EQUIPMENT) {
                     $qualification->setAttributeNodeNS(new \DOMAttr('type', 'equipment'));
                     $qualification->setAttributeNodeNS(new \DOMAttr('description', 'Car'));
-                }
-                $qualification->setAttributeNodeNS(new \DOMAttr('category', $qualificationModel->getCategory()));
-                if ($qualificationModel->getExperience() == Qualification::EXPERIENCE_NOT_REQUIRED) {
-                    $qualification->setAttributeNodeNS(new \DOMAttr('yearsOfExperience', '1'));
-                } elseif ($qualificationModel->getExperience() == Qualification::EXPERIENCE_REQUIRED) {
+                } elseif ($qualificationModel->getType() == Qualification::TYPE_EXPERIENCE) {
+                    $qualification->setAttributeNodeNS(new \DOMAttr('type', strtolower($qualificationModel->getType())));
```

Comment:
We don't want our internal constants to reflect the api.

```suggestion
                    $qualification->setAttributeNodeNS(new \DOMAttr('type', 'experience'));
```